crates/context/config/src/client.rs

```diff
@@ -228,12 +231,16 @@ impl Client<AnyTransport> {
                         eyre::bail!("missing account id for `{}` signer", network);
                     };
 
+                    let access_key: PrivateKeySigner =
+                        PrivateKeySigner::from_str(&credentials.secret_key.clone())
```

Comment:
shouldn't need to clone here

```suggestion
                        PrivateKeySigner::from_str(&credentials.secret_key)
```